library/src/main/java/com/otaliastudios/zoom/ZoomEngine.kt

```diff
@@ -428,8 +428,18 @@ internal constructor(context: Context) : ViewTreeObserver.OnGlobalLayoutListener
         mTransformationGravity = gravity
     }
 
-    override fun setSmallerPolicy(@SmallerPolicy policy: Int) {
-        mSmallerPolicy = policy
+    /**
+     * Sets the content alignment. Can be any of the constants defined in [Alignment].
+     * The content will be aligned and forced to the specified side of the container.
+     * Defaults to [Alignment.CENTER].
```

Comment:
Well `ZoomApi.ALIGNMENT_DEFAULT` which is set to `Alignment.CENTER`... don't really know if that's important but if it's ever going to change it will likely be forgotten to be changed in this documentation part.